templates/web/smidsy/about/faq.html

```diff
@@ -3,95 +3,174 @@
-    </dl>
-    
-    <h2><a name="organisation"></a>Organisation Questions</h2>
-    <dl>
-        <dt>Who built Collideoscope?</dt>
-        <dd>Collideoscope was built by <a href="http://www.mysociety.org/">mySociety</a> and <a href="http://itpworld.net">Integrated Transport Planning Ltd</a>. It is based upon the FixMyStreet project, <a href="http://www.fixmystreet.com/faq#organisation">See full credits for FixMyStreet</a>.
-        </dd>
-
-        <dt>Why build Collideoscope?</dt>
-        <dd>
-            <p>This website is our attempt to enable society to collaborate in order to collect some much-needed evidence to inform the ongoing cycle-safety debate. Our aim is to build a map of cycling-related incidents and near-misses for the UK so that transport planners, health researchers and decision-makers can see for the first time where incidents are happening, how often they take place, and how serious they are. We believe this kind of evidence fills a gap that will potentially enable local Councils to make proactive changes to highway environments so as to prevent serious, and potentially fatal, cycling incidents from occurring in the future.</p>
-        </dd>
-    </dl>
+
+<h2 name="general">General</h2>
+
+<dl>
+    <dt>What is this site for?</dt>
+    <dd>Collideoscope invites you to report cycling collisions and near misses in the UK. It collects the reports together and makes the data available to planners, researchers and campaigners with the aim of making our roads safer for all.</dd>
+
+    <dt>Who is behind this site?</dt>
+    <dd>It’s a joint project from <a href="https://www.mysociety.org">mySociety</a> and the <a href="https://www.no-excuses.org.uk">Merseyside Road Safety Partnership</a>, and it’s powered by mySociety’s long-running <a href="https://fixmystreet.org">FixMyStreet platform</a>.</dd>
+</dl>
+
+<h2 name="reporting">Making a report</h2>
+
+<dl>
+    <dt>What kind of reports can be made on Collideoscope?</dt>
+    <dd>You can report actual collisions between a bicycle and another vehicle, or near misses where an incident was averted. Additionally you may report incidents where no other vehicle was involved, such as collisions or near misses with pedestrians, street furniture... or anything else.</dd>
+    <dd>Please <strong>do not report incidents requiring immediate medical or police attention</strong> — or at least, not until you have requested help via the emergency services’ official channels.</dd>
+
+    <dt>Where do reports go?</dt>
+    <dd>Reports are sent to the local authority with responsibility for the area in which the incident occured. Additionally, the data is published on this site and is available to anyone who might like to use it, such as researchers, town planners, campaigners and local residents. <a href="/reports">Find out more.</a></dd>
```

Comment:
I don't like full stops being part of the link, is that just me?

Reply:
IIRC @MyfanwyNixon’s preference is also for "whole sentence" links to include the full stop. So I think we’re keeping this.